### ISSUE TITLE
[Question] Why no support for Debian 11 ARM64?

### ISSUE THREAD

**Issue by rassie** [microsoft/playwright#18573]
I've stumbled over this problem while dealing with custom Docker images for our development. They are based on `node:16-bullseye-slim`, which is Debian 11 at the core. However, there seems to be no support in `install-deps` for Debian 11 ARM64, so that on x86 systems `playwright install-deps` succeeds, but on ARM64 fails, because it's recognized as `generic-linux-arm64`, which is more or less a Ubuntu 20.04 fallback, which in turns tries installing `ttf-ubuntu-font-family`, which is in the `non-free` section on Debian and thus not available to `install-deps` on Debian 11.

Long story short: it seems that at least for Chromium generic binaries are used for Debian 11 and the other binaries could be at least compatible between Ubuntu 22.04 and Debian 11. Is there a deeper reason why Debian 11 ARM64 binaries are not available for Playwright?

**Comment by mxschmitt** [comment]
There is a feature request for it here: https://github.com/microsoft/playwright/issues/18319

**Comment by aslushnikov** [comment]
Closing in favor of #18319. Please upvote to help us prioritize!
